transmart-server/grails-app/conf/logback.groovy

```diff
@@ -0,0 +1,90 @@
+import ch.qos.logback.classic.encoder.PatternLayoutEncoder
+import ch.qos.logback.core.ConsoleAppender
+import ch.qos.logback.core.FileAppender
+import ch.qos.logback.core.rolling.FixedWindowRollingPolicy
+import ch.qos.logback.core.rolling.RollingFileAppender
+import ch.qos.logback.core.rolling.SizeBasedTriggeringPolicy
+import grails.util.BuildSettings
+import grails.util.Environment
+
+// See http://logback.qos.ch/manual/groovy.html for details on configuration
+appender('stdout', ConsoleAppender) {
+    encoder(PatternLayoutEncoder) {
+        pattern = "%level %logger - %msg%n"
+    }
+}
+
+root(WARN, ['stdout'])
+
+boolean productionMode = Environment.current == Environment.PRODUCTION
+def logDirectory = BuildSettings.TARGET_DIR
+if (productionMode) {
+    def catalinaBase = System.getProperty('catalina.base') ?: '.'
+    logDirectory = "${catalinaBase}/logs".toString()
+}
+if ((productionMode || Environment.isDevelopmentMode()) && logDirectory) {
+    appender("FULL_STACKTRACE", FileAppender) {
+        file = "${logDirectory}/stacktrace.log"
+        append = true
+        encoder(PatternLayoutEncoder) {
+            pattern = "%level %logger - %msg%n"
+        }
+    }
+    logger("StackTrace", ERROR, ['FULL_STACKTRACE'], false)
+}
+if (productionMode && logDirectory) {
+    appender('transmart', RollingFileAppender) {
+        file = "${logDirectory}/transmart.log"
+        encoder(PatternLayoutEncoder) {
+            pattern = '%d{dd-MM-yyyy HH:mm:ss,SSS} %5p %c{1} - %m%n'
+        }
+        triggeringPolicy(SizeBasedTriggeringPolicy) {
+            maxFileSize = '100MB'
+        }
+        rollingPolicy(FixedWindowRollingPolicy) {
+            fileNamePattern = "${logDirectory}/transmart.%i.log"
+            minIndex = 1
+            maxIndex = 9
+        }
+    }
+    root(WARN, ['transmart'])
+}
+
+/**
+ * Configuration for writing audit metrics.
```

Comment:
This block is a part of audit metrics recording configuration, described on [transmartfoundation wiki](https://wiki.transmartfoundation.org/display/transmartwiki/Recording+audit+metrics). Originally it was a part of  
[Config.groovy](https://github.com/transmart/transmartApp/blob/master/grails-app/conf/Config.groovy#L206,) (also commented out by default)
but since Log4j has been replaced by grails-app/conf/logback.groovy for logging, it was moved here. I am not sure if it will be still used (outside of transmartApp), so I did not remove it, just in case.